app/MMB/src/scenes/timeline/event/Event.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/mobile-shared';
+
+import IconVerticalLine, {
+  type IconVerticalLineType,
+} from './IconVerticalLine';
+import TimeLocation, { type TimeLocationType } from './TimeLocation';
+import DescriptionComment, {
+  type DescriptionCommentType,
+} from './DescriptionComment';
+
+export type Props = {|
+  ...IconVerticalLineType,
+  ...TimeLocationType,
+  ...DescriptionCommentType,
+  first: boolean,
+|};
```

Comment:
This is a really ugly type. Do not reuse `Props` please. It does not make any sense. One way how to make it better is to compose the components instead.